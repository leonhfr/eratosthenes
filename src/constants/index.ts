// Definition.
export const BUCKET_GEOJSON = 'scenicroutes-geojson';
