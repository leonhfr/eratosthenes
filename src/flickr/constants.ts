// Definition.
export const API_ENDPOINT = 'https://api.flickr.com/services/rest/';

export const API_PARAMETERS = { format: 'json' };
