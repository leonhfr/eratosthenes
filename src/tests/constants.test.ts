import * as constants from '../constants';

// Code.
describe('constants', () => {
  it('should expose the expected entries', () => {
    expect(constants).toMatchSnapshot();
  });
});
