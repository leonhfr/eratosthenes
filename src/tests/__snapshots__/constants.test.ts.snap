// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should expose the expected entries 1`] = `
Object {
  "BUCKET_GEOJSON": "scenicroutes-geojson",
}
`;
